docarray/array/storage/elastic/backend.py

```diff
@@ -46,6 +46,7 @@ class ElasticConfig:
     ef_construction: Optional[int] = None
     m: Optional[int] = None
     columns: Optional[Union[List[Tuple[str, str]], Dict[str, str]]] = None
+    root_id: bool = False
```

Comment:
Maybe we should just always store the root_id and not even ask for this config. The overhead seems small (compared to the complete serialized docs that we always store), and it would improve the user experience. What do you think @AnneYang720 @alaeddine-13 @JoanFM ?

Reply:
Yes, store by default